test/femdriver.jl

```diff
@@ -1,42 +1,43 @@
 module FEMDriver
 
-using Test
```

Comment:
The code in this file can be already uncommented.

The API of the new version of Gridap is not exactly the same as the old one, but it is similar. See an example here:

https://github.com/gridap/Gridap.jl/blob/master/test/GridapTests/PoissonTests.jl

Essentially, you will need tho change
-  `FESpace` -> `TestFESpace`
- `LinearFEOperator` -> `AffineFEOperator`
- `free_dofs` -> `get_fee_dofs`